src/index.html

```diff
@@ -1,13 +1,446 @@
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a 
+                  href="tel:+1 234 5555-55-55" 
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a 
+                  href="mailto:hello@miami.com" 
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Adress
+                </div>
+                <a 
+                  href="https://goo.gl/maps/YX3CpAfsBMsfAYKA7" 
+                  class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="#" method="get">
+                <label for="name">
+                  <input 
+                    name="name" 
+                    type="text"
+                    placeholder="Name"
+                    class="contact-us__field form-field"
+                    required
+                  >
+                </label>
+                <label for="email">
+                  <input 
+                    name="email" 
+                    type="email"
+                    placeholder="Email"
+                    class="contact-us__field form-field"
+                    required
+                  >
+                </label>
+               
+                <textarea 
+                  name="message"
+                  placeholder="Type message here..."
+                  class="
+                  contact-us__field 
+                  contact-us__field--last
+                  form-field 
+                  form-field--textarea"
+                  required
+               ></textarea>
+
+                <button 
+                  type="reset"
```

Comment:
```suggestion
                  type="submit"
```
Це має бути сабміт

Дизайн самої кнопки відрізняється від макету. Фон не повинен бути сірим
<img width="469" alt="Снимок экрана 2022-02-16 в 17 51 47" src="https://user-images.githubusercontent.com/66570142/154303145-fc98e68d-b745-4bbf-b238-b3263bd3d13e.png">